pkg/debugapi/balances.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+	"sort"
+
+	"net/http"
+)
+
+type balanceResponse struct {
+	Peer    string `json:"peer"`
+	Balance int64  `json:"balance"`
+}
+
+type balancesResponse struct {
+	Balances []balanceResponse `json:"balances"`
+}
+
+func (s *server) balancesHandler(w http.ResponseWriter, r *http.Request) {
+
+	balances, err := s.Accounting.Balances()
+
+	if err != nil {
+		jsonhttp.InternalServerError(w, "Can not get balances")
```

Comment:
i missed this, but this error could be extracted to a separate `var` declaration in the beginning of the package, then you can use the error when returning the error here. you can then expose this error in `export_test.go` and verify the error inside the test using the actual error, and not against the hardcoded string which asserts the error now